bootstrap/src/bootstrap/rocket-mvd-resources.ts

```diff
@@ -22,14 +22,32 @@ import { SimpleGlobalization } from '../i18n/simple-globalization'
 declare var window: { ZoweZLUX: typeof ZoweZLUXResources,
                       COM_RS_COMMON_LOGGER: Logger};
 window; /* Suppress TS error */
+
+// This is ithe core logger
 let logger = new Logger();
 logger.addDestination(logger.makeDefaultDestination(true,true,true,true,true));
 window.COM_RS_COMMON_LOGGER = logger;
 
+// component logger
+export var bootstrapLogger : ZLUX.ComponentLogger = logger.makeComponentLogger("_zsf.bootstrap");
+
+fetch('/ZLUX/plugins/org.zowe.zlux.bootstrap/web/assets/i18n/log/messages_en.json')
```

Comment:
A thorough test with all files missing is a great idea, also a test with an invalid JSON would be good. As far as behavior I have observed: code and parameters printing but no message, which is decent.
2020-02-24 20:01:52.061 <ZWED:460084> TS3215 INFO (org.zowe.zlux.bootstrap:adminnotificationdata,native array.js:958) ZWED5001I GET

Reply:
It does not appear that a missing file or invalid JSON can cause Zowe to crash but I would like to investigate this behavior a little bit further.  Again, will comment back when approved.

Reply:
@timgerstel Hey, I know this won't crash. This was set in stone during @1000TurquoisePogs 's recent changes that added % indicators for missing plugins. Instead, those plugins will be glossed over, it will be logged which plugins failed to load, and Zowe will continue to run, or it will try to unless a core plugin failed to load, then the server will continue to run but the desktop won't etc. etc.